internal/consensus/common_test.go

```diff
@@ -388,6 +388,24 @@ func subscribeToVoter(cs *State, addr []byte) <-chan tmpubsub.Message {
 	return ch
 }
 
+func subscribeToVoterBuffered(cs *State, addr []byte) <-chan tmpubsub.Message {
+	votesSub, err := cs.eventBus.Subscribe(context.Background(), testSubscriber, types.EventQueryVote)
+	if err != nil {
+		panic(fmt.Sprintf("failed to subscribe %s to %v", testSubscriber, types.EventQueryVote))
+	}
```

Comment:
Since this is a test, perhaps consider:

```suggestion
func subscribeToVoterBuffered(t *testing.T ,cs *State, addr []byte) <-chan tmpubsub.Message {
	t.Helper()
	votesSub, err := cs.eventBus.Subscribe(context.Background(), testSubscriber, types.EventQueryVote)
	if err != nil {
		t.Fatalf("failed to subscribe %s to %v", testSubscriber, types.EventQueryVote)
	}
```

Reply:
Be aware also that when you go to merge this with master, the API for subscribers has changed.